extensions/ipynb/src/notebookSerializer.ts

```diff
@@ -81,7 +81,7 @@ export class NotebookSerializer implements vscode.NotebookSerializer {
 		const notebookContent: Partial<nbformat.INotebookContent> = data.metadata?.custom || {};
 		notebookContent.cells = notebookContent.cells || [];
 		notebookContent.nbformat = notebookContent.nbformat || 4;
-		notebookContent.nbformat_minor = notebookContent.nbformat_minor || 2;
+		notebookContent.nbformat_minor = notebookContent.nbformat_minor ?? 2;
```

Comment:
a Bug, if the value was `0` we were replacing with `2`

Reply:
Same for the above? Maybe that can't happen